<launch>
	<node name="calculate_global_marker_client" pkg="calculate_global_marker" type="calculate_global_marker_client" respawn="false" output="screen">
	</node>
<!--
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find calcalate_global_marker)/rviz/axis.rviz">
	</node>
	-->
</launch>
